pkg/fakeoperator/fakeoperator.go

```diff
@@ -47,6 +52,7 @@ func init() {
 	utilruntime.Must(apiextensionsv1.AddToScheme(scheme))
 	utilruntime.Must(fakev1alpha1.AddToScheme(scheme))
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	utilruntime.Must(certv1alpha2.AddToScheme(scheme))
```

Comment:
Why do you need this exactly?

Reply:
yeah, I agree with @nmiculinic that this is not needed

Reply:
yeah, we don't need it